t/44-scripts.t

```diff
@@ -33,4 +33,9 @@ for my $script (sort keys %types) {
       or diag "Output: $out";
 }
 
+subtest 'ovs' => sub {
```

Comment:
The 44-scripts.t is only for generic testing of all our scripts. Anything more specific should go into its own test file.

Reply:
Ok, I've created a new test `46-openvswitch.t`